src/Library/demos/Button/main.blp

```diff
@@ -0,0 +1,45 @@
+using Gtk 4.0;
+using Adw 1;
+
+ Adw.StatusPage{
+  title: "Button";
+  description: "A simple way to trigger an event";
+
+  Box {
+     orientation: vertical;
+
+     Box {
+      orientation: vertical;
+      halign: center;
+
+      Button test_button {
+        label: "Active Button";
+        margin-bottom: 30;
+        styles ["pill","flat"]
+      }
+
+      Button disabled_button {
+        label: "Disabled Button";
+        margin-bottom: 30;
+        sensitive: false;
+        styles ["pill","destructive-action"]
+      }
+
+      LinkButton{
+        label: "Tutorial";
+        uri: "https://developer.gnome.org/documentation/tutorials/beginners/components/button.html";
+    }
+
+      LinkButton{
+        label: "API Reference";
+        uri: "https://docs.gtk.org/gtk4/class.Button.html";
+    }
+
+      LinkButton{
+        label: "Human Interface Guidelines";
+        uri: "https://developer.gnome.org/hig/patterns/controls/buttons.html";
+    }
+
+     }
```

Comment:
please take care of the indentation until it's automated

it should be 2 spaces